app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.java

```diff
@@ -26,28 +28,24 @@ public LicenseFragmentHelper(@Nullable final Activity activity) {
         weakReference = new WeakReference<>(activity);
     }
 
-    private static String getFinishString(final Activity activity) {
-        return activity.getApplicationContext().getResources().getString(R.string.finish);
-    }
-
     /**
      * @param context the context to use
      * @param license the license
      * @return String which contains a HTML formatted license page
      * styled according to the context's theme
      */
-    public static String getFormattedLicense(final Context context, final License license) {
+    private static String getFormattedLicense(final Context context, final License license) {
         if (context == null) {
-            throw new NullPointerException("context is null");
+            throw new IllegalArgumentException("context is null");
         }
         if (license == null) {
-            throw new NullPointerException("license is null");
+            throw new IllegalArgumentException("license is null");
         }
```

Comment:
This is also unrelated to dependency updates, but this block of code should be replaced by `@NotNull` parameters, or just removed alltoghether, because a NullPointerException would be thrown anyway crashing the app, so it makes no sense to check for nullity and then do not do anything different.

Reply:
That's why I made it an `IllegalArgumentException`, though making it `@NotNull` is probably better indeed